src/actions/motion.ts

```diff
@@ -740,12 +740,16 @@ class MoveRepeatReversed extends BaseMovement {
 }
 
 @RegisterAction
-class MoveLineEnd extends BaseMovement {
+class MoveDownLineEnd extends BaseMovement {
```

Comment:
I think it's better to keep this as `MoveLineEnd`. That's the functionality that most people associate with this key.